rskj-core/src/test/java/co/rsk/peg/FederationTestUtils.java

```diff
@@ -138,6 +139,39 @@ public static void spendFromErpFed(
 //        System.out.println(Hex.toHexString(spendTx.bitcoinSerialize()));
     }
 
+    public static void addSignatures(Federation federation, List<BtcECKey> signers, BtcTransaction tx){
+        Script fedInputScript = createBaseInputScriptThatSpendsFromTheFederation(federation);
+        tx.getInput(0).setScriptSig(fedInputScript);
+
+        Sha256Hash sighash = tx.hashForSignature(
+            0,
+            federation.getRedeemScript(),
+            BtcTransaction.SigHash.ALL,
+            false
+        );
+
+        int totalSigners = signers.size();
+        int requiredSignatures = totalSigners / 2 + 1;
+
+        for (int i = 0; i < requiredSignatures; i++) {
+            BtcECKey privateKey = signers.get(i);
+            BtcECKey publicKey = BtcECKey.fromPublicOnly(privateKey.getPubKeyPoint().getEncoded(true));
+
+            BtcECKey.ECDSASignature sig = privateKey.sign(sighash);
+            TransactionSignature txSig = new TransactionSignature(sig, BtcTransaction.SigHash.ALL, false);
```

Comment:
```suggestion
            BtcECKey.ECDSASignature signature = privateKey.sign(sighash);
            TransactionSignature txSig = new TransactionSignature(signature, BtcTransaction.SigHash.ALL, false);
```
Short For sigHash or signature?

Reply:
Updated